include/wx/translation.h

```diff
@@ -145,6 +145,11 @@ class WXDLLIMPEXP_BASE wxTranslations
     wxString GetBestTranslation(const wxString& domain,
                                 const wxString& msgIdLanguage = "en");
 
+    // find best and all other suitable translation languages for given domain
+    wxArrayString GetAllGoodTranslations(const wxString& domain, wxLanguage msgIdLanguage);
```

Comment:
I'm not sure what is "good" really supposed to mean (and it kind of implies that we also have "bad" translations, which is not really the case, is it?). Maybe call this `GetAcceptableTranslations()` or maybe `GetAllUsableTranslations()`?

Reply:
> I'm not sure what is "good" really supposed to mean (and it kind of implies that we also have "bad" translations, which is not really the case, is it?). 

"Good" comes from the idea that since there is a best translation (as in the pre-existing `GetBestTranslation()`), all the candidates are therefore "good". And one of the good is the best. (Though IMHO, the "best" term is somewhat ambiguous too; the "best" translation is chosen best merely because it exists, not because of e.g. its completeness or high quality.) 

I also considered calling this function `GetBestTranslations()`, but the one-letter difference to another function name could be too hard to spot. I also considered leaving this function undocumented.

> Maybe call this `GetAcceptableTranslations()` or maybe `GetAllUsableTranslations()`?

Without looking at the docs, those (too) sound quite similar in behaviour to `GetAvailableTranslations()`, which already exists. By looking at function names only, it would not be clear why translations that are "available" aren't "acceptable" or "usable" (or "good").

Reply:
FWIW I think "acceptable" is a but more precise than "good" as it implies that there is some criterion of acceptability (in this case, it's whether the language is understood by the user or not). But yes, it's a relatively minor difference... If you really like "good" and if nobody else shares my dissatisfaction with using a so generic term, we could leave it like this.

Reply:
Sorry, after agonizing about this for a (too) long time, I've finally changed the name as "good" really doesn't sound good to me.

Reply:
Thank you for merging this PR, and sorry for me not having the time to add an example to the internat sample yet. I plan to do so, though, in near future, in a new PR. And the `GetAcceptableTranslations()` naming is fine by me.